### PR TITLE
[CDAP-1831] Fix explore on hive 0.10 by using a 'use db' statement rathe...

### PR DESCRIPTION
**Pull request by bdmogal**
...r than SessionState.setCurrentDatabase which is unavailable in hive 0.10

Jira: [CDAP-1831](https://issues.cask.co/browse/CDAP-1831)
Build: http://builds.cask.co/browse/CDAP-RBT167-1

### CODE REVIEW

**Comment by anew** [comment]
Is there a unit test?
Other than that LGTM

File: cdap-explore/src/main/java/co/cask/cdap/explore/service/hive/HiveCDH4ExploreService.java [hunk 1]
```diff
@@ -105,6 +109,22 @@ protected OperationHandle doExecute(SessionHandle sessionHandle, String statemen
   }
 
   @Override
+  protected void setCurrentDatabase(String dbName) throws IOException, HiveSQLException, ExploreException {
```
Comment (ParthGandhi) [inline cdap-explore/src/main/java/co/cask/cdap/explore/service/hive/HiveCDH4ExploreService.java]: Just make this `throws Exception`

**Comment by ParthGandhi** [comment]
One nit for uniformity. Otherwise LGTM.

@anew Yes, there are unit tests for this.

**Comment by bdmogal** [comment]
We don't seem to have tests that use hive 0.10. There are tests that would call the ``setCurrentDatabaseMethod``. But none  that use hive 0.10. We should probably have a way of testing hive 0.10, since we support it.

**Comment by bdmogal** [comment]
Maybe a much larger effort though?

**Comment by bdmogal** [comment]
However, the way this problem was discovered was we ran audi on a CDH 4.3 cluster that has hive 0.10. Maybe we should make that part of our regular audi runs as well.

File: cdap-explore/src/main/java/co/cask/cdap/explore/service/hive/HiveCDH4ExploreService.java (hunk 1 above)
Comment (ParthGandhi) [inline cdap-explore/src/main/java/co/cask/cdap/explore/service/hive/HiveCDH4ExploreService.java]: Ignore ^

**Pull request merged**